src/mote/templates/mainpage.html

```diff
@@ -51,18 +52,31 @@
                 <div class="collapse navbar-collapse" id="navbarSupportedContent">
                     <ul class="navbar-nav me-auto mb-2 mb-lg-0">
                         <li class="nav-item">
-                            <a class="nav-link"
-                               aria-current="page"
-                               data-bs-toggle="modal"
-                               data-bs-target="#chanmode"
-                               type="button"
-                               onclick="populate_channel_list();">
-                                Channels
-                            </a>
                         </li>
                     </ul>
                     <div class="d-flex">
-                        <a style="color:#676767; text-decoration:none;" href="https://getfedora.org/" target="_blank" type="button">Get Fedora</a>
+                    <a class="nav-link"
+                    aria-current="page"
+                    data-bs-toggle="modal"
+                    data-bs-target="#chanmode"
+                    type="button"
+                    onclick="populate_channel_list();">
+                    Meetings by Channel
+                    </a>
+                    <a class="nav-link"
+                    aria-current="page"
+                    data-bs-toggle="modal"
+                    data-bs-target="#chanmode"
+                    type="button"
+                    onclick="populate_recent_meeting_list();">
```

Comment:
I think we can remove the `onclick` attribute for now, as the `meetings by group` page doesn't exist yet.